### ISSUE TITLE
[BUG] Use the Page Title as the Alt Text for Past Paper Preview Images

### ISSUE THREAD

**Issue by Aminnaderi10** [GamaEdtech/front#1004]
Currently, all past paper preview images use the same fixed alt text: "Psat Paper Lesson".

To improve accessibility and provide more descriptive image metadata, the alt text should be dynamically populated from the title of the corresponding past paper page. This will ensure that each preview image has a unique and relevant alt attribute instead of a generic value.
@Farimah71
